src/main/java/Parser.java

```diff
@@ -0,0 +1,71 @@
+import java.util.Scanner;
+
+/**
+ *  makes sense of the user command
+ */
+public class Parser {
+
+    Scanner scan;
+    Ui ui = new Ui();
+
+    public Parser() {
+        this.scan = new Scanner(System.in);
+    }
+
+    /**
+     * scanner scan task's instruction and run accordingly
+     * @throws DukeException when users input task which bot does not understand
+     */
+    public void scan() throws DukeException {
+        ui.greeting();
+        String command = scan.nextLine();
+        while (!command.equals("bye")) {
+            if (command.contains("find")) {
```

Comment:
Potential bugs with using 'contains' method: what happens if the user input is "todo find"?